R/drive_download.R

```diff
@@ -2,8 +2,10 @@
 #'
 #' @template file
 #' @param out_path Character. Path for output file. Will default to its Google Drive
-#'   name. Google files must include the intended extension and can be downloaded to types
+#'   name. Google files must include the intended extension. This can be specified by the
+#'   `out_path` or the `type` parameter. Google files can be downloaded to types
 #'   specified in the [Drive API documentation](https://developers.google.com/drive/v3/web/manage-downloads).
+#' @param type Character. Extension you would like for the local file.
```

Comment:
right now I use it to append to the file name, so it must be an extension, but I could change that. Do you think it'd be worthwhile to create an inverse function for `drive_mime_type()` that takes an accepted mime type or extension and returns the extension?